.golangci.yaml

```diff
@@ -24,7 +24,7 @@ linters-settings:
       - performance
       - style
   gocyclo:
-    min-complexity: 10
+    min-complexity: 15
```

Comment:
the additional `if` statement triggered the cyclomatic complexity linter. It's currently configured at 10, which is quite strict: default is 50, [with a recommendation of 10-20](https://golangci-lint.run/usage/linters/#gocyclo), so raising this to 15 which is still in line with the recommendation.